tools/walletextension/main/cli.go

```diff
@@ -20,17 +20,23 @@ const (
 	nodeRPCWebsocketAddressName    = "nodeRPCWebsocketAddress"
 	nodeRPCWebsocketAddressDefault = "testnet.obscu.ro:13001"
 	nodeRPCWebsocketAddressUsage   = "The address on which to connect to the node via RPC using websockets"
+
+	logPathName    = "logPath"
+	logPathDefault = ""
```

Comment:
shouldn't it log by default in like a :"wallet-extension.log" file, in the current folder

Reply:
Addressed. I chose `wallet_extension_logs.txt`, for consistency with the enclave and host logs.